src/components/search/CustomMarker.tsx

```diff
@@ -26,6 +26,26 @@ const CustomMarker = (props: CustomMarkerProps) => {
   const hovered = id === hoveredId;
   const map = useMapContext();
 
+  const { clusters } = useClusterContext();
+  const isMarkerInCluster = !!clusters.find(
+    (cluster) => cluster.length > 1 && cluster.find((pin) => pin.id === id)
```

Comment:
Do we need the `cluster.length > 1` check?

Reply:
The `clusters` array can have nested nested arrays of length one, which indicates that a marker isn't part of a cluster and is rendered normally.
So some sort of check to make sure the marker isn't in a cluster of length 1 is required to make sure.